rt/ws/security/src/main/java/org/apache/cxf/ws/security/sts/provider/SecurityTokenService.java

```diff
@@ -73,7 +73,7 @@ RequestSecurityTokenResponseCollectionType issue(
               partName = "response")
     @Action(input = "http://docs.oasis-open.org/ws-sx/ws-trust/200512/RST/Issue",
             output = "http://docs.oasis-open.org/ws-sx/ws-trust/200512/RSTRC/IssueFinal")
-    @WebMethod(operationName = "Issue")
+    @WebMethod(operationName = "IssueSingle")
```

Comment:
The operation was introduced manually (see please d3c73f2af973a875febe3bc6f3f34c251fa5339b), not part of the WS-Trust spec